shipcat_filebacked/src/manifest.rs

```diff
@@ -184,11 +184,11 @@ impl ManifestSource {
             image: Some(self.build_image(&base.name)?),
 
             version: overrides.version.build(&())?,
-            kong: defaults.kong.build(&KongBuildParams {
+            kong_apis: defaults.kong.build(&KongBuildParams {
                 service: base.name.to_string(),
                 region: region.clone(),
                 hosts: overrides.hosts,
-            })?.unwrap_or(None),
+            })?.unwrap_or_default().map_or(Default::default(), |k| vec![k]),
```

Comment:
woah, dense line. feels like one `default` invocation here is redundant?

Reply:
presumably this is building "a default" and then vectorising it at the end?

Reply:
```suggestion
            })?.unwrap_or_default().map(|k| vec![k]).unwrap_or_default(),
```

is an alternative.

Unfortunately, `KongSource.build()` needs to return an option, because `kong: { host: null, uris: null }` is equivalent to `kong.enabled: false` for backwards compatibility.

https://github.com/Babylonpartners/shipcat/blob/master/shipcat_filebacked/src/kong.rs#L56

Reply:
If we can migrate everyone to this new map based way of definining all of the APIs, we should be able to get rid of that check and just make it invalid to omit `host`/`uris`.

Reply:
Ah, double Option. Very well. I think your suggestion reads slightly better in that case, just because `map_or` is one of those you hardly ever see.